excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/parser/ListFieldTypeParser.java

```diff
@@ -81,6 +84,14 @@ public ListFieldTypeParser(ParseContext parseContext) {
         parseContext.addToAllTypes(fixedListType);
         result.addNew(fixedListType);
 
+        // Add as DynamicList with no list items as it will be dynamically added by the services
+        final FieldTypeEntity dynamicListType = new FieldTypeEntity();
+        dynamicListType.setBaseFieldType(fixedListBaseType);
```

Comment:
shouldn't this be:
`dynamicListType.setBaseFieldType(dynamicListBaseType);`
?

Reply:
correct thanks for spotting it